src/Symfony/Component/Form/Resources/translations/sf_form.en.xlf

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0"?>
+<xliff version="1.2" xmlns="urn:oasis:names:tc:xliff:document:1.2">
+    <file source-language="en" datatype="plaintext" original="file.ext">
+        <body>
+            <trans-unit id="1">
+                <source>No choice available</source>
+                <target>![CDATA[<em>No choice available.</em>]]</target>
```

Comment:
the cdata tags have syntax errors. correct is: `<![CDATA[` and `]]>` (not escaped)